Screenshots tests are failing

### What
- Screenshots tests are failing
### Part of
- #4167 

### Logs/Stacktrace

- [ ] https://github.com/openfoodfacts/openfoodfacts-androidapp/blob/develop/app/src/androidTestScreenshots/java/openfoodfacts/github/scrachx/openfood/AbstractScreenshotTest.kt
- [ ] e: /home/runner/work/openfoodfacts-androidapp/openfoodfacts-androidapp/app/build/generated/source/kapt/offPlaystoreScreenshotsAndroidTest/openfoodfacts/github/scrachx/openfood/AbstractScreenshotTest_MembersInjector.java:13: error: cannot find symbol
- [ ] @QualifierMetadata("dagger.hilt.android.qualifiers.ApplicationContext")
- [ ]  ^
- [ ]   symbol: class QualifierMetadata
- [ ] https://github.com/openfoodfacts/openfoodfacts-androidapp/blob/develop/app/src/androidTest/java/openfoodfacts/github/scrachx/openfood/repositories/TaxonomiesRepositoryTest.kt
- [ ] e: /home/runner/work/openfoodfacts-androidapp/openfoodfacts-androidapp/app/build/generated/source/kapt/offPlaystoreScreenshotsAndroidTest/openfoodfacts/github/scrachx/openfood/repositories/TaxonomiesRepositoryTest_MembersInjector.java:13: error: cannot find symbol
- [ ] @QualifierMetadata("dagger.hilt.android.qualifiers.ApplicationContext")
- [ ]  ^
- [ ]   symbol: class QualifierMetadata
- [ ] w: warning: The following options were not recognized by any processor: '[eventBusIndex, kapt.kotlin.generated]'
- [ ] 
- [ ] > Task :app:kaptOffPlaystoreScreenshotsAndroidTestKotlin FAILED

--- a/app/build.gradle.kts
+++ b/app/build.gradle.kts
@@ -177,8 +177,10 @@ dependencies {
     androidTestUtil("androidx.test:orchestrator:1.4.1")
 
     // Hilt for Android Testing
-    androidTestImplementation("com.google.dagger:hilt-android-testing:2.41")
-    kaptAndroidTest("com.google.dagger:hilt-android-compiler:2.41")
+    androidTestImplementation("com.google.dagger:hilt-android-testing:2.39.1")
+    kaptAndroidTest("com.google.dagger:hilt-android-compiler:2.39.1")
+    androidTestImplementation("com.google.dagger:dagger:2.39.1")
+    kaptAndroidTest("com.google.dagger:dagger-compiler:2.39.1")
 
     androidTestImplementation("androidx.test:runner:1.3.0") { exclude("junit") }
     androidTestImplementation("androidx.test:rules:1.4.0")

--- a/app/src/androidTestScreenshots/java/openfoodfacts/github/scrachx/openfood/test/ScreenshotActivityTestRule.kt
+++ b/app/src/androidTestScreenshots/java/openfoodfacts/github/scrachx/openfood/test/ScreenshotActivityTestRule.kt
@@ -7,7 +7,7 @@ import androidx.preference.PreferenceManager
 import androidx.test.platform.app.InstrumentationRegistry
 import androidx.test.rule.ActivityTestRule
 import openfoodfacts.github.scrachx.openfood.utils.LocaleManager
-import openfoodfacts.github.scrachx.openfood.utils.PreferencesManager
+import openfoodfacts.github.scrachx.openfood.utils.PreferencesService
 import org.apache.commons.lang3.StringUtils
 import org.junit.Assert
 
@@ -28,7 +28,7 @@ class ScreenshotActivityTestRule<T : Activity?>
     override fun beforeActivityLaunched() {
         try {
             runOnUiThread {
-                PreferencesManager(context).isFirstTimeLaunch = firstTimeLaunched
+                PreferencesService(context).isFirstTimeLaunch = firstTimeLaunched
                 localeManager.saveLanguageToPrefs(
                         InstrumentationRegistry.getInstrumentation().targetContext,
                         screenshotParameter!!.locale,

--- a/app/src/androidTestScreenshots/java/openfoodfacts/github/scrachx/openfood/test/ScreenshotParameter.kt
+++ b/app/src/androidTestScreenshots/java/openfoodfacts/github/scrachx/openfood/test/ScreenshotParameter.kt
@@ -2,5 +2,4 @@ package openfoodfacts.github.scrachx.openfood.test
 
 import java.util.*
 
-
 data class ScreenshotParameter(val locale: Locale, var productCodes: List<String>)

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -20,7 +20,7 @@ buildscript {
     val kotlinVersion by extra("1.5.30")
     val jacksonVersion by extra("2.12.3")
     val greendaoVersion by extra("3.3.0")
-    val hiltVersion by extra("2.38.1")
+    val hiltVersion by extra("2.39.1")
     repositories {
         google()
         mavenCentral()

